frontend/src/containers/MasterWindow.js

```diff
@@ -177,20 +181,20 @@ class MasterWindowContainer extends PureComponent {
       tabId: activeTabId,
     });
 
-    const tabLayout =
-      master.layout.tabs &&
-      master.layout.tabs.filter((tab) => tab.tabId === activeTabId)[0];
-
-    const orderBy = tabLayout.orderBy;
+    const tabLayout = getTab(master.layout, activeTabId);
+    const orderBy = tabLayout ? tabLayout.orderBy : null;
```

Comment:
what happens if orderBy is undefined ?

Reply:
Nothing, we won't set the sorting order when requesting data.